components/script/dom/window.rs

```diff
@@ -2218,13 +2229,19 @@ impl Window {
         assert!(self.document.get().is_none());
         assert!(document.window() == self);
         self.document.set(Some(document));
-        if !self.unminify_js {
-            return;
+        if self.unminify_js {
+            // Set a path for the document host to store unminified scripts.
+            let mut path = env::current_dir().unwrap();
+            path.push("unminified-js");
+            *self.unminified_js_dir.borrow_mut() =
+                Some(path.into_os_string().into_string().unwrap());
+        } else if self.unminify_css {
```

Comment:
This makes unminifying css and js exclusive. Let's have two separate if statements instead